src/Catrobat/Services/CatrobatCodeParser/Scripts/WhenBGChangeScript.php

```diff
@@ -25,14 +25,14 @@ protected function create()
 
     if (count($this->script_xml_properties->look) != 0)
     {
-      if ($this->script_xml_properties->look[Constants::REFERENCE_ATTRIBUTE] != null)
+      if ($this->script_xml_properties->look[Constants::REFERENCE_ATTRIBUTE] !== null)
       {
         $this->look_file_name = $this->script_xml_properties->look
-          ->xpath($this->script_xml_properties->look[Constants::REFERENCE_ATTRIBUTE])[0]->fileName;
+          ->xpath($this->script_xml_properties->look[Constants::REFERENCE_ATTRIBUTE])[0]['fileName'];
       }
       else
       {
-        $this->look_file_name = $this->script_xml_properties->look->fileName;
+        $this->look_file_name = $this->brick_xml_properties->look['fileName'];
```

Comment:
`brick_xml_properties` does not exist in class `Script`. Must be changed to `script_xml_properties`.